docker/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -40,8 +40,6 @@
     default_candidates.append("stun:8555")
 
     go2rtc_config["webrtc"] = {"candidates": default_candidates}
-else:
-    print("[INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually")
```

Comment:
Hey, instead of removing you can simply print to stderr which it should be since my implementation. I'm bad at Python, but the following should work:


sys.stderr.write("[INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually\n")